docs/advanced-usage/connecting-cloud-storage/gcp.md

```diff
@@ -0,0 +1,41 @@
+Integrations can be established to Google Cloud Storage.
+
+To get started, ensure you have admin access within Kolena.
+Navigate to your [Organization Settings's Integration Tab](https://app.kolena.io/redirect/organization?tab=integrations) and click "Add Integration", then "GCP".
+
+### 1. Save Integration to Create a Service Account
+
+From the [Integration Tab](https://app.kolena.io/redirect/organization?tab=integrations), saving the GCP integration will create a service account.
+Upon creation, the integration's `client_email` will be used to provide Kolena permission to load data from your GCP buckets.
+
+### 2. Grant Service Account Read Access
+
+Within your GCP console, navigate to the bucket that contains your images.
+Click on the permissions tab.
+Click the "Grant Access" button and grant the service account created in [step 1](#1-save-integration-to-create-a-service-account) the `Storage Object Viewer` role.
+The `Storage Object Viewer` role offers the following permissions:
+
+- Grants access to view objects and their metadata, excluding ACLs.
+- Grants access to list the objects in a bucket.
+
+​ ​
+
+### 3. Provide CORS Access
+
+Create a json file `cors.json` with the following content:
+
+```json
+[
+  {
+    "origin": ["https://app.kolena.io/<your-organization>"],
```

Comment:
Does path have any effect for `origin`?

Reply:
I've adapted this from notion so I'm not 100% https://www.notion.so/kolena/Kolena-GCP-Access-88706fb85e074ec1be278d067689cd60

Are you thinking this could be `"origin": ["https://app.kolena.io"]` ?

Reply:
I think if the path does have an effect it's not important. Will update to `"https://app.kolena.io"`